.github/workflows/woke.yml

```diff
@@ -0,0 +1,18 @@
+# yamllint disable rule:line-length
+# name: woke
```

Comment:
Without `name`, the status field is called `.github/workflows/woke.yml / woke (pull_request)`
Is there some reason you have `# name: woke` commented out?

Reply:
No, I don't think so... Anyway, I updated the top-level name as you suggested in https://github.com/linux-system-roles/crypto_policies/pull/54#issuecomment-1351675147, which is much better. Thanks!

> Check for non-inclusive language / woke (pull_request)